code/modules/projectiles/guns/specialist.dm

```diff
@@ -13,9 +13,23 @@
 	var/aimed_shot_cooldown
 	var/aimed_shot_cooldown_delay = 2.5 SECONDS
 
+	var/enable_aimed_shot_laser = TRUE
+	var/sniper_lockon_icon = "sniper_lockon"
+	var/obj/effect/ebeam/sniper_beam_type = /obj/effect/ebeam/laser
+	var/sniper_beam_icon = "laser_beam"
+
+/obj/item/weapon/gun/rifle/sniper/get_examine_text(mob/user)
+	. = ..()
+	if(!has_aimed_shot)
+		return
+	var/dat = ""
+	dat += SPAN_NOTICE("This weapon has an unique ability, Aimed Shot, allowing it to deal great damage after a windup. ")
+	dat += SPAN_BOLDNOTICE("Additionally, the aimed shot can be sped up with a tracking laser, which is enabled by default but may be disabled.")
+	. += dat
```

Comment:
genuinely schizo code, make it 1 line normally

Reply:
bro i copied it from YOUR lever action code

Reply:
yeaaaaa